cpp/test/sg/decisiontree_batchedlevel_unittest.cu

```diff
@@ -116,7 +121,7 @@ class BatchedLevelAlgoUnitTestFixture {
     input.nSampledRows = n_row;
     input.nSampledCols = n_col;
     input.rowids = row_ids;
-    input.nclasses = 0;  // not applicable for regression
+    input.nclasses = 1;  // not applicable for regression
```

Comment:
minor nitpick: is this now a limitation that nclasses can't be zero, even for regression?

Reply:
Yes, this is enforced by a check in builder. We will have multioutput regression in the near future and this will make more sense. I will change this variable to numOutputs or similar.